code/index.html

```diff
@@ -1,22 +1,77 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="utf-8" />
-    <title>Project 2: The News Site</title>
-    <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
-  </head>
-  <body>
-    <header>
-      <!-- Some kind of logo and navbar -->
-    </header>
-
-    <section class="big-news">
-      <!-- The big news content -->
-    </section>
-
-    <section class="">
-      <!-- The other news  -->
-    </section>
-  </body>
-</html>
+
+<head>
+  <meta charset="utf-8" />
+  <title>Project 2: Good stuff</title>
+  <link rel="stylesheet" href="./style.css" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link href="https://fonts.googleapis.com/css?family=Roboto:300,900&display=swap" rel="stylesheet">
+</head>
+
+<body>
+  <header>
+    <div class="hamburger">
+      <div></div>
+      <div></div>
+      <div></div>
+    </div>
+    <img src="goodstuff-logo.svg" alt="goodstuff-logo">
+    <nav>
+      <a href="#">Cookies</a>
+      <a href="#">Food</a>
+      <a href="#">Cakes</a>
+      <a href="#">Bread</a>
+    </nav>
+  </header>
+
+  <section class="big-news">
+    <img src="images/scones_3.jpg" alt="scones image">
```

Comment:
You don't need "image" as a part of the `alt`, it will have "image" describe to it because it is an image element 
```suggestion
    <img src="images/scones_3.jpg" alt="scones">
```